tsadar/inverse/loss_function.py

```diff
@@ -9,6 +9,7 @@
 import equinox as eqx
 
 from ..core.thomson_diagnostic import ThomsonScatteringDiagnostic
+from tsadar.core.modules import exchange_params, get_filter_spec
```

Comment:
can you use the relative paths here like on the previous line? That helps with downstream use as an installable package that provides classes